apps/studio/components/interfaces/Auth/BasicAuthSettingsForm/BasicAuthSettingsForm.tsx

```diff
@@ -244,7 +247,7 @@ const BasicAuthSettingsForm = () => {
                     }}
                     formValues={values}
                   />
-                  {isProPlanAndUp ? (
+                  {isProPlanAndUp || !IS_PLATFORM ? (
```

Comment:
Shouldn't isProPlanAndUp be false all the time anyway given `isSuccessSubscription` should never be true? 🤔

Reply:
Yeah it's false, but we actually need this branch to run, because we don't want to prompt for a Pro upgrade on local. I added a new variable `promptProPlanUpgrade` to make that clearer.